doc/classes/KinematicBody2D.xml

```diff
@@ -53,21 +53,21 @@
 			<return type="bool">
 			</return>
 			<description>
-				Returns [code]true[/code] if the body is on the ceiling. Only updates when calling [method move_and_slide].
+				Returns [code]true[/code] if the body is on the ceiling. Only updates when it calls [method move_and_slide].
 			</description>
 		</method>
 		<method name="is_on_floor" qualifiers="const">
 			<return type="bool">
 			</return>
 			<description>
-				Returns [code]true[/code] if the body is on the floor. Only updates when calling [method move_and_slide].
+				Returns [code]true[/code] if the body is on the floor. Only updates when it calls [method move_and_slide].
 			</description>
 		</method>
 		<method name="is_on_wall" qualifiers="const">
 			<return type="bool">
 			</return>
 			<description>
-				Returns [code]true[/code] if the body is on a wall. Only updates when calling [method move_and_slide].
+				Returns [code]true[/code] if the body is on a wall. Only updates when it calls [method move_and_slide].
 			</description>
```

Comment:
As per my previous comment, all of these changes to "it calls" don't make sense. It's unclear what "it" refers to.

Reply:
Note that "Only updates when [method move_and_slide] is called." is the correct way to convert passive to active voice here.

Reply:
@bojidar-bg  Actually, "is called" is still the passive voice. However, I agree that changing it to the past tense would be better. I would then further suggest dropping the "Only".

Furthermore, if it is being changed, it should also be corrected, since it is updated by both `move_and_slide()` and `move_and_slide_with_snap()`

Reply:
Thank you for the replies. I'll revert back any unnecessary changes and add in some of your suggestions, such as the addition of move_and_slide_with_snap() on Line 70, in a new commit.

Reply:
To be consistent, if you're modifying line 70, lines 20, 27, 49, 56 and 63 should all be modified in the same way, and the same changes should be made to the KinematicBody3D.xml file too.

Reply:
What about line 36, it has [method move and slide] as well. Should I add [method move and slide with snap]?

Reply:
Yes, good spot.